### PR TITLE
create user added

### PR DESCRIPTION
**Pull request by YogiSoni-lazy**

### CODE REVIEW

File: modules/install/pages/admin.adoc [hunk 1]
```diff
@@ -1,24 +1,161 @@
...
-== Creating Administrator Accounts
+* JBoss EAP 8 has multiple subsystems that you can customize to support Jakarta EE application requirements, such as database connectivity, batch processing, and integration tools.
 
-* Talk about the *add-user* script and other ways to create accounts
-* Distinguish between management users and application users
-* CSV file format
-* Accessing the JBoss EAP web console
-* Accessing the JBoss EAP CLI
-* Do not go too much into details about EAP CLI or web console here. Remember there is a separate chapter for EAP config...
+* The subsystem configuration is maintained in the `standalone.xml` file for the standalone mode, and the `domain.xml` file for a managed domain mode.
```
Comment (rsriniva) [inline modules/install/pages/admin.adoc]: You have not yet introduced what a subsystem is. Replace this with generic sounding "Jboss EAP configuration"

File: modules/install/pages/admin.adoc [hunk 2]
```diff
@@ -1,24 +1,161 @@
...
 
-== References
+* These configuration files can be customized using three different approaches:
 
-* Add links to product docs
+** **Management console**: As a system administrator, you can manage most of the capabilities of your standalone or managed domain deployments by using a browser.
```
Comment (rsriniva) [inline modules/install/pages/admin.adoc]: s/browser/JBoss EAP web console or Management Console

File: modules/install/pages/admin.adoc [hunk 3]
```diff
@@ -1,24 +1,161 @@
...
+** **Management Command Line Interface(CLI)**:
+The CLI offers a management model for viewing and modifying attributes and for performing operations, on a standalone server or a managed domain.
 
-. Step 1
-. Step 2
-.. Step 2.a
-.. Step 2.b
-. Step 3
+** **Edit XML configuration files manually**:
+The settings of a standalone server or a managed domain are maintained in XML-based configuration files, which can be modified directly.
```
Comment (rsriniva) [inline modules/install/pages/admin.adoc:17]: Add a WARNING that editing XML files directly should be avoided as much as possible and that either the CLI or the web console is preferred method

File: modules/install/pages/admin.adoc [hunk 4]
```diff
@@ -1,24 +1,161 @@
...
+Is this correct yes/no? **yes**
+Added user 'newadmin' to file '/opt/EAP-8.0.0/standalone/configuration/mgmt-users.properties'
+Added user 'newadmin' to file '/opt/EAP-8.0.0/domain/configuration/mgmt-users.properties'
+Added user 'newadmin' with groups  to file '/opt/EAP-8.0.0/standalone/configuration/mgmt-groups.properties'
+Added user 'newadmin' with groups  to file '/opt/EAP-8.0.0/domain/configuration/mgmt-groups.properties'
+----
+
+** Log in to the management console using web browser to http://localhost:9990/, as the `newadmin` user and `redhat123` as the password. You should see the home page of the JBoss EAP 8 management console.
+
+* Create a new application user named `appuser` and password `redhat123`.
```
Comment (rsriniva) [inline modules/install/pages/admin.adoc]: No need to show this here. We will show them to do this later in the course. We cannot show anything related to app users here. remove this section

**Review by rsriniva — CHANGES_REQUESTED** [review]
Excellent. short and sweet! minor comments to be addressed

File: modules/install/pages/admin.adoc (hunk 2 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/admin.adoc]: Changed the wordings

File: modules/install/pages/admin.adoc (hunk 1 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/admin.adoc]: Updated

File: modules/install/pages/admin.adoc (hunk 3 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/admin.adoc:17]: Added the warning

File: modules/install/pages/admin.adoc (hunk 4 above)
Comment (YogiSoni-lazy) [reply modules/install/pages/admin.adoc]: removed the section

**Review by rsriniva — APPROVED** [review]
Approved. Please merge

**Pull request merged**